polygon/bridge/bridge.go

```diff
@@ -63,10 +61,12 @@ type Bridge struct {
 	borConfig                    *borcfg.BorConfig
 	eventFetcher                 eventFetcher
 	stateReceiverContractAddress libcommon.Address
+
+	reader *Reader
```

Comment:
Could you rename this to `eventReader` and the struct to `EventReader`. I think this gives a clearer indication of  what we are reading.

Reply:
It is not just an event reader, we can get the block number from transaction hash too. I think `EventReader` is a misnomer. 

`Reader` contains all the read-only operations for the bridge. Since it is part of the `bridge` package, it will be referenced as `bridge.Reader` outside the package, which I think is clear.

Reply:
Ah, I see. I think it is `func (r *Reader) EventTxnLookup(ctx context.Context, borTxHash libcommon.Hash) (uint64, bool, error) {` that confused me. Could you add a short description for it?